[2018-10-02 20:32:15] [    INFO] --- Setup logging. Log file: ./output/s1_p_2_2_3/CIFAR10/optimizer_name=Adam/no_cuda=True/seed=42/02-10-18-20h-32m-15s/logs.txt (setup.py:511)
[2018-10-02 20:32:15] [    INFO] --- Arguments saved to: ./output/s1_p_2_2_3/CIFAR10/optimizer_name=Adam/no_cuda=True/seed=42/02-10-18-20h-32m-15s/args.txt (setup.py:514)
[2018-10-02 20:32:15] [    INFO] --- Initialize Tensorboard SummaryWriter (setup.py:519)
[2018-10-02 20:32:16] [ WARNING] --- Git status is ignored! (RunMe.py:205)
[2018-10-02 20:32:16] [    INFO] --- Model CNN_basic expects input size of (32, 32) (image_classification.py:60)
[2018-10-02 20:32:16] [    INFO] --- Loading CIFAR10 from:datasets/CIFAR10 (setup.py:233)
[2018-10-02 20:32:16] [    INFO] --- Verifying the dataset integrity - quick (dataset_integrity.py:182)
[2018-10-02 20:32:18] [    INFO] --- Newly measured timestamp: Tue Oct  2 20:27:15 2018 (dataset_integrity.py:188)
[2018-10-02 20:32:18] [   ERROR] --- The dataset has been modified. The last_modified field does not match: old[Tue Oct  2 20:23:10 2018] new[Tue Oct  2 20:27:15 2018] (dataset_integrity.py:194)
[2018-10-02 20:32:18] [    INFO] --- Dataset loaded as images (setup.py:267)
[2018-10-02 20:32:18] [    INFO] --- Setting up model CNN_basic (setup.py:86)
[2018-10-02 20:32:18] [    INFO] --- Loading weights for data balancing (setup.py:101)
[2018-10-02 20:32:18] [    INFO] --- Begin training (image_classification.py:73)
[2018-10-02 20:32:23] [    INFO] --- val   epoch[-1]: Acc@1=10.000	Loss=2.3071	Batch time=0.022 (0.002 to load data) (evaluate.py:151)
[2018-10-02 20:32:48] [    INFO] --- val   epoch[0]: Acc@1=47.560	Loss=1.4484	Batch time=0.024 (0.002 to load data) (evaluate.py:151)
[2018-10-02 20:33:14] [    INFO] --- val   epoch[1]: Acc@1=53.900	Loss=1.3034	Batch time=0.023 (0.002 to load data) (evaluate.py:151)
[2018-10-02 20:33:40] [    INFO] --- val   epoch[2]: Acc@1=57.930	Loss=1.1875	Batch time=0.025 (0.002 to load data) (evaluate.py:151)
[2018-10-02 20:34:09] [    INFO] --- val   epoch[3]: Acc@1=59.930	Loss=1.1426	Batch time=0.042 (0.003 to load data) (evaluate.py:151)
[2018-10-02 20:34:38] [    INFO] --- val   epoch[4]: Acc@1=59.270	Loss=1.1677	Batch time=0.029 (0.002 to load data) (evaluate.py:151)
[2018-10-02 20:34:44] [    INFO] --- test  epoch[4]: Acc@1=59.450	Loss=1.1532	Batch time=0.031 (0.002 to load data) (evaluate.py:151)
[2018-10-02 20:34:44] [    INFO] --- Training completed (image_classification.py:91)
[2018-10-02 20:34:44] [    INFO] --- Time taken for train/eval/test is: 0:02:27 (RunMe.py:257)
